core/arch/arm/plat-vexpress/main.c

```diff
@@ -52,9 +52,9 @@ void main_init_gic(void)
 	vaddr_t gicd_base;
 
 	gicc_base = (vaddr_t)phys_to_virt(GIC_BASE + GICC_OFFSET,
-					  MEM_AREA_IO_SEC);
+					  MEM_AREA_IO_SEC, GIC_DIST_REG_SIZE);
```

Comment:
s/GIC_DIST_REG_SIZE/GIC_CPU_REG_SIZE/

Reply:
It is mapped as GIC_DIST_REG_SIZE, but ok, suppose it can be fixed later.